Fitness/Controllers/HomeController.swift

```diff
@@ -308,7 +316,9 @@ extension HomeController: NavigationDelegate {
         classListViewController.calendarDateSelected = classListViewController.calendarDatesList[3]
         classListViewController.calendarCollectionView.reloadData()
         classListViewController.getClassesFor(date: classListViewController.calendarDateSelected)
-        classListViewController.filterOptions(params: FilterParameters())
+        if let _ = classListViewController.currentFilterParams {
```

Comment:
when you do `if let _ = ...` it is better to do `if classListViewController.currentFilterParams != nil`